plugins/search/src/components/SearchModal/SearchModal.tsx

```diff
@@ -37,12 +37,12 @@ import CloseIcon from '@material-ui/icons/Close';
 import React, { useCallback, useEffect, useRef } from 'react';
 import { useNavigate } from 'react-router-dom';
 
-import { rootRouteRef } from '../../plugin';
+import { rootRouteRef, SearchResultChildrenProvider } from '../../plugin';
 
 /**
  * @public
  */
-export interface SearchModalChildrenProps {
+export interface SearchModalChildrenProps extends SearchResultChildrenProvider {
```

Comment:
It's not necessary to forward the search results here, because people who define a children function are following the third use case, they are completely customizing the modal.
```suggestion
export interface SearchModalChildrenProps {
```

Reply:
Oh I get it now